packages/settings/src/schema/central.ts

```diff
@@ -8,8 +8,9 @@ import {
   questionCodeIdsDescription,
 } from './definitions';
 import { extractDefaults } from './utils';
+import type { SettingsSchema } from '../types';
 
-export const centralSettings = {
+export const centralSettings: SettingsSchema = {
```

Comment:
Specifying the type here makes VSCode / typescript report errors earlier and in context, e.g. misspellings get red squiggles immediately on the keys.

Reply:
Nope can't do that lol that breaks the types. Hilarious type system.